affiliated/registry.json

```
@@ -143,7 +143,7 @@
             "name": "astroML",
             "maintainer": "Jake Vanderplas and Brigitta Sipőcz <bsipocz@uw.edu>",
```

let's use a permanent email for this
```suggestion
            "maintainer": "Jake Vanderplas and Brigitta Sipőcz <brigitta.sipocz@gmail.com>",
```